internal/distro/rhel9/images.go

```diff
@@ -346,6 +346,9 @@ func imageInstallerImage(workload workload.Workload,
 
 	img.SquashfsCompression = "xz"
 
+	// put the kickstart file in the root of the iso
+	img.ISORootKickstart = true
```

Comment:
What's the ideal? I remember there was a discussion with the installer team, but forgot the outcome. If not for compat, where should we be putting the .ks?

Reply:
For this PR, I'm making sure to not make any changes to image definitions, but we should think about this and decide what we want.

Reply:
https://hackmd.io/hwMbx1CXQRKvHXGW8yQTMg